example.html

```diff
@@ -12,13 +12,25 @@
       let beamsClient;
 
       PusherPushNotifications.init({
-        instanceId: 'deadc0de-2ce6-46e3-ad9a-5c02d0ab119b',
+        instanceId: '1b880590-6301-4bb5-b34f-45db1c5f5644',
```

Comment:
Did you mean to commit the changes to this file?

Reply:
It was just for testing purposes. We can revert this change if you want.